scripts/accumulator.crt

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
As discussed by skype, this file will be generated on-the-fly during test harness, so it needs to be removed from repo.